prow/flagutil/github.go

```diff
@@ -40,6 +44,24 @@ func (o *GitHubOptions) AddFlags(fs *flag.FlagSet) {
 	fs.Var(&o.endpoint, "github-endpoint", "GitHub's API endpoint (may differ for enterprise).")
 	fs.StringVar(&o.TokenPath, "github-token-path", "/etc/github/oauth", "Path to the file containing the GitHub OAuth secret.")
 	fs.StringVar(&o.deprecatedTokenFile, "github-token-file", "", "DEPRECATED: use -github-token-path instead.  -github-token-file may be removed anytime after 2019-01-01.")
+	o.baseURL = getBaseURL(o.endpoint)
+
+}
+
+// getBaseURL infer base url from the github endpoint
+func getBaseURL(endpoints Strings) string {
+	for _, endpoint := range endpoints.vals {
+		endpointURL, err := url.Parse(endpoint)
+		if err != nil {
```

Comment:
Don't ignore the error please. Either return the error, log.Fatal and/or panic

Reply:
@fejta - Didn't we have validation for endpoint urls before ? I was expecting to fallback on it, but anyway double checking can't do harm.